front/src/styles/index.scss

```diff
@@ -1,86 +1,86 @@
-:global{
-  #root{
-    height:100%;
-
-    & > div, & > div > div{
-      height: 100%;
-    }
-  }
-
-}
-*, *:before, *:after {
-  box-sizing: inherit;
-}
-html {
-  box-sizing: border-box;
-  font-size: 62.5%;
-
-}
-
-body {
-  overflow: hidden !important;
-  background-color: #f6f5f5;
-  position: absolute;
-  width: 100%;
-  height: 100%;
-  padding: 0;
-  margin: 0;
-  font-family: Lato, sans-serif;
-  font-size: 1.6rem;
-  -webkit-touch-callout: none;
-
-  * {
-    -webkit-tap-highlight-color: rgba(255, 255, 255, 0);
-  }
-}
-
-/* Setting default font family, font size, font weight, color for all placeholders*/
-a, abbr, address, article, aside, audio, b, blockquote, body, caption, cite,
-code, dd, del, dfn, dialog, div, dl, dt, em, fieldset, figure, footer, form, h1,
-h2, h3, h4, h5, h6, header, hgroup, hr, html, i, iframe, img, ins, kbd, label,
-legend, li, mark, menu, nav, object, ol, p, pre, q, samp, section, small, span,
-strong, sub, sup, table, tbody, td, tfoot, th, thead, time, tr, ul, var, video {
-  padding: 0;
-  margin: 0;
-  border: 0;
-  outline: 0;
-}
-
-*, *:before, *:after {
-  box-sizing: border-box;
-  padding: 0;
-  margin: 0;
-}
-
-h1, h2, h3, h4, h5, h6, label, p, button, abbr, a, span, small {
-  font-smoothing: antialiased;
-  font-weight: normal;
-  -webkit-font-smoothing: antialiased;
-  text-size-adjust: 100%;
-}
-
-h2{
-  font-size: 30px;
-}
-
-h3{
-  font-size: 25px;
-}
-
-h4{
-  font-size: 20px;
-}
-
-h5{
-  font-size: 15px;
-}
-
-h6{
-  font-size: 14px;
-}
-
-a {
-  color: #8a8888;
-  text-decoration: none;
-  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
-}
+:global{
+  #root{
+    height:100%;
+
+    & > div, & > div > div{
+      height: 100%;
+    }
+  }
+
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+
+}
+
+body {
+  overflow: auto !important;
```

Comment:
I changed only this line. I don't know why there is a problem with end line chars...